src/request/export/index.html

```diff
@@ -213,19 +213,19 @@ <h1 data-i18n="export-file-intro-heading" class="nq-h1">Save your account</h1>
                                 <use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-skull"/>
                             </svg>
                             <span data-i18n="export-file-intro-accident">
-                                As a part of your browser's data it might be deleted by accident.
+                                It might be deleted automatically or by accident.
```

Comment:
I think we should clarify here, that it is not _us_ who is deleting it automatically, but the operating system and browser.

Reply:
I don't think that makes a difference here. Obviously, we are not harmfully deleting user data on purpose. At the same time, we have built this system that comes with these dangers and deficiencies, so we are exposing a user to this risk. At the same time, with the launch of the new KG, there is the possibility that it is, in fact, us that might be deleting data. Let's stick to this clear message: It might be deleted, doesn't matter by whom.